test/plots/rectanglePlotTests.ts

```diff
@@ -39,6 +39,44 @@ describe("Plots", () => {
       VERIFY_CELLS((<any> rectanglePlot)._renderArea.selectAll("rect"));
       svg.remove();
     });
+
+    it("retrieves the correct entity under a point", () => {
+      var xScale = new Plottable.Scales.Linear();
+      var yScale = new Plottable.Scales.Linear();
+      var svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+      var dataset = new Plottable.Dataset(DATA);
+      var plot = new Plottable.Plots.Rectangle()
+        .x((d) => d.x, xScale).x2((d) => d.x2)
+        .y((d) => d.y, yScale).y2((d) => d.y2)
+        .addDataset(dataset).renderTo(svg);
+      var entities = (<Plottable.Plots.Rectangle<number, number>> plot).entitiesAt({ x: 140, y: 140 });
+      assert.lengthOf(entities, 1, "a single entity is retrieved at (140, 140)");
+      assert.strictEqual(entities[0].index, 2, "entity retrieved is at index 2");
+      svg.remove();
+    });
+
+    it("retrieves correct entities under a point", () => {
```

Comment:
The first one is a single entity retrieved; the second one is multiple entities.